src/components/DataVisuals/index.js

```diff
@@ -0,0 +1,68 @@
+import { Typography, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+import Carousel from "react-multi-carousel";
+
+import useStyles from "./useStyles";
+
+import DataVisualCard from "@/pesayetu/components/DataVisualCard";
+import Section from "@/pesayetu/components/Section";
+import "react-multi-carousel/lib/styles.css";
+
+const responsive = {
+  tablet: {
+    breakpoint: { max: 1279, min: 768 },
+    items: 2,
+  },
+};
+
+const DataVisuals = ({ title, items, ...props }) => {
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+  const classes = useStyles(props);
```

Comment:
Let's use `<Hidden>` instead with `implementation=css`. Its less line of codes, and `css media query` over `js media query` anytime!

`<Hidden only="md" implementation="css"></Hidden>`  //screen other than tablet
`<Hidden smDown lgUp implementation="css">` //for tablet

Reply:
Done. Thanks for the pointer